pkg/event/event.go

```diff
@@ -61,7 +61,6 @@ type Metadata struct {
 // Source represents client information who triggered the event.
 type Source struct {
 	Host      string `json:"host"`
-	Port      string `json:"port"`
```

Comment:
Does this removal cause backward compatibility problem in target servers/services?

Reply:
This is not part of S3 spec, so I thought it won't cause problems. Infact this value has been empty for many releases and no users have reported yet.

Reply:
Expanded question is currently `"source"` JSON field in `"Records"` of event contains  `"Port"`.  Is this removal causes backward compatibility?

Reply:
I already answered that, practically speaking this value has been empty, because this value is not populated in any of our notifications for perhaps more than a year.

Just having empty value doesn't make sense anymore, but just for backward compatiblity sake I can keep the struct as is and never populate this value.

What do you recommend ?

Reply:
Resolving as stale.